src/frontend/src/common/utils/samples.ts

```diff
@@ -1,2 +1,6 @@
 export const getLineageFromSampleLineages = (lineages: Lineage[]): Lineage =>
   lineages?.[0];
```

Comment:
do we still need this?

Reply:
Yes, it is used in the lineages column definition in the samples tables.